README.md

```diff
@@ -1,2 +1,462 @@
 # SQL Style Guide
 Guide for formatting SQL scripts
+
+# Why Style Guide
+
+1. Increase readability
+2. Be consistent
+
+# Guidelines with examples
+
+## General 
+
+- Use all lowercase SQL
+
+```
+-- Good
+select distinct order_merchant_id as store_id, member_id
+from ebates_prod.dw.order_transactions
+
+-- Bad : Dont use upper case & lower case combinations 
+SELECT DISTINCT order_merchant_id AS store_id, member_id
+FROM ebates_prod.dw.order_transactions
+
+-- Bad : Dont use upper case for queries 
+SELECT DISTINCT ORDER_MERCHANT_ID AS STORE_ID, MEMBER_ID
+FROM EBATES_PROB.DW.ORDER_TRANSACTIONS
+```
+
+- Fieldnames or Variable names or aliased table names should be in `snake_cased`
+
+```
+-- Good
+select count(*) as item_count
+
+-- Bad
+select count(*) as itemCount
+
+-- Bad
+select count(*) as itemcount
+```
+
+- For Indentation, use 4 spaces for one level of indentation and 8 for deeper and so on..
+- Lines of SQL should be no longer than 100 characters. TODO : How do we make sure of that? 
+- No trailing white space
+- If queries/models are run by DBT then do not mention schema or database name, just use table name
+- Use `!=` instead of `<>` since it is more popular in other programming languages
+- Ordering and grouping by a number (eg. GROUP BY 1, 2) is preferred
```

Comment:
Actually I don't like ordering or grouping with numbers... 

I can only accept `group by 1`. Probably `group by 1, 2`. 

But often time I see people write `group by 1,2,3,4` or even `group by 1,3,5,7`. and it is super error prone and hard to read.